### PR TITLE
add unicode-msgpack as a serializer for all queue_util uses

### PR DESCRIPTION
**Pull request by bobbyrenwick**
@EDITD/hackers this is the final piece in being able to send msgpack messages between things in the pipe. It means we will specify `'unicode-msgpack'` when saying what the serializer will be for a message.

### CODE REVIEW

File: queue_util/__init__.py [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+def unpack(s):
+    return msgpack.unpackb(s, encoding='utf-8')
+
+register(
+    'unicode-msgpack',
+    pack,
+    unpack,
+    content_type='application/x-unicode-msgpack',
+    content_encoding='binary'
+)
```
Comment (pkatseas) [inline queue_util/__init__.py]: As we talked about in person, I think we should move this to a separate module (cleaner) and import it here so we make sure it's run and registered.

**Comment by bobbyrenwick** [comment]
LGTP!

**Comment by bobbyrenwick** [comment]
Thanks @pkatseas!

**Comment by pkatseas** [comment]
Does LGTM

**Pull request merged**